content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+  applicant's card.
+- :guilabel:`Attendees`: Select the people who should be in attendance. The default employee listed
+  is the person who is creating the meeting. Add as many other people as desired.
+
+Meeting details tab
+^^^^^^^^^^^^^^^^^^^
+
+- :guilabel:`Starting At`: Using the drop-down calendar module, select the date and time for the
+  meeting start time.
+- :guilabel:`Ending At`: Using the drop-down calendar module, select the date and time for the
+  meeting end time.
+- :guilabel:`Duration`: This field auto populates based on the :guilabel:`Starting At` and
+  :guilabel:`Ending At` times entered. If the meeting time is adjusted, this field automatically
+  adjusts to the correct duration length.
+- :guilabel:`All Day`: If the meeting is an all day long event, check the box next to :guilabel:`All
+  Day`. After this box is checked, the :guilabel:`Duration` field is hidden from view, a well as the
+  meeting start and end times.
+- :guilabel:`Organizer`: The employee who created the meeting is populated in this field. Use the
+  drop-down menu to change the selected employee.
+- :guilabel:`Description`: Enter a brief description in this field. There is an option to enter
+  formatted text, such as numbered lists, headings, tables, as well as links, photos, and more. Type
+  `/` and a list of options are presented. Scroll through the options and click on the desired item.
+  The item appears in the field and can be modified. Each command presents a different pop-up
+  window. Follow the instructions for each command to complete the entry.
+- :guilabel:`Reminders`: Select a reminder from the drop-down menu. Options include
+  :guilabel:`Notification`, :guilabel:`Email`, and :guilabel:`SMS Text Message`, each with a
+  specific time period before the event (hours, days, etc). The reminder chosen will alert the
+  meeting participants of the meeting via the selected option.
+- :guilabel:`Location`: Enter the location for the meeting.
+- :guilabel:`Meeting URL`: If the meeting is virtual, enter the meeting link in this field.
+- :guilabel:`Tags`: Select any tag(s) for the meeting using the drop-down menu. There is no limit to
+  the number of tags that can be used.
+
+Options tab
+^^^^^^^^^^^
+
+- :guilabel:`Recurrent`: If the meeting should repeat at a selected interval (not typical for a
+  first interview), check the box next to :guilabel:`Recurrent`. This reveals a recurrence module.
+  Enter the details for how often the meeting should repeat, including an end time.
+- :guilabel:`Privacy`: Select the visibility of this meeting. Options are :guilabel:`Public`,
+  :guilabel:`Private`, and :guilabel:`Only internal users`. :guilabel:`Public` allows for everyone
+  to see the meeting, :guilabel:`Private` allows only the attendees listed on the meeting to see the
+  meeting, and :guilabel:`Only internal users` allows everyone in the company to see the meeting.
```

Comment:
Portal users? I'm not sure where to view this... I don't think we have this posted on the website... I may need to reach out to a PO on this one.... unless you know the permissions.

Reply:
I had checked with Quentin on this, and I edited this so its the 'logged into the company's DB' which I think sounds better. If you are logged into the system, you can see it.